runtime/themes/voxed.toml

```diff
@@ -0,0 +1,110 @@
+type = "light-blue"
+"type.builtin" = "functionish" # TODO: distinguish?
+constructor = "lilac"
+function = "functionish"
+"function.macro" = "lilac"
+"function.builtin" = "typeish" 
+tag = "almond"
+comment = "bgray"
+constant = "tan"
+"constant.builtin" = "#D38588"
+#string = "#B8BB26" or gruvgreen -- Green if you prefer it for strings
+string = "redish"
+"constant.numeric" = "functionish"
+"constant.character.escape" = "honey"
+# used for lifetimes
+label = "honey"
+
+"markup.heading" = "functionish"
+"markup.list" = "status-two"
+"markup.quote" = "tan"
+"markup.bold" = { fg = "sglow", modifiers = ["bold"] }
+"markup.italic" = { fg = "sglow", modifiers = ["italic"] }
+"markup.strikethrough" = { modifiers = ["crossed_out"] }
+"markup.link.url" = { fg = "sglow", modifiers = ["underlined"] }
+"markup.link.text" = "greenish"
+"markup.raw" = "light-grey"
+
+"diff.plus" = "#7DDF64"
+"diff.minus" = "#F22B29"
+"diff.delta" = "#6f44f0"
+
+# TODO: diferentiate doc comment
+# concat (ERROR) @error.syntax and "MISSING ;" selectors for errors
+
+# NOTE: bg is empty so it is transparent on the terminal
+# if you want the bg to not be transparent rename give bg
+# the same value as fg and remove fg :)
+"ui.background" = { fg = "#25262B", bg = "#25262B" }
+"ui.background.separator" = { fg = "sglow" }
+"ui.linenr" = { fg = "light-grey", modifiers = ["italic"] }
+"ui.linenr.selected" = { fg = "bpink", modifiers = ["bold"] }
+"ui.statusline" = { fg = "status-two", bg = "light-grey", modifiers = ["bold", "dim"] }
+"ui.statusline.inactive" = { fg = "black", bg = "bgray-two" }
+"ui.popup" = { fg = "bgray", bg = "" }
+"ui.window" = { fg = "white" }
+"ui.help" = { bg = "#3f4047", fg = "light-grey" }
+
+"ui.text" = { fg = "lavender" }
+"ui.text.focus" = { fg = "maize", bg = "bgray" }
+"ui.text.inactive" = "sirocco"
+"ui.virtual" = { fg = "comet" }
+
+"ui.virtual.indent-guide" = { fg = "comet" }
+
+"ui.selection" = { bg = "maize" }
+"ui.selection.primary" = { fg = "white", bg = "bgray" }
+# TODO: namespace ui.cursor as ui.selection.cursor?
+"ui.cursor.select" = { bg = "delta" }
+"ui.cursor.insert" = { bg = "white" }
+"ui.cursor.match" = { fg = "#212121", bg = "#6C6999" }
+"ui.cursor" = { modifiers = ["reversed"] }
+"ui.cursorline.primary" = { bg = "bossanova" }
+"ui.highlight" = { bg = "bossanova" }
+"ui.highlight.frameline" = { bg = "#634450" }
+"ui.debug" = { fg = "#634450" }
+"ui.debug.breakpoint" = { fg = "apricot" }
+"ui.menu" = { fg = "white", bg = "#23232d" }
+"ui.menu.selected" = { fg = "white", bg = "bgray" }
+"ui.menu.scroll" = { fg = "lavender", bg = "comet" }
+
+"diagnostic.hint" = { underline = { color = "silver", style = "curl" } }
+"diagnostic.info" = { underline = { color = "delta", style = "curl" } }
+"diagnostic.warning" = { underline = { color = "lightning", style = "curl" } }
+"diagnostic.error" = { underline = { color = "apricot", style = "curl" } }
+
+warning = "bpink"
+error = "bsienna"
+info = "maize"
+hint = "tan"
+
+[palette]
```

Comment:
The palette doesn't define many of the colors that are used in the theme, for example `apricot`, `lavendar` or `sorocco`